qiskit/providers/ibmq/api/rest/job.py

```diff
@@ -15,11 +15,15 @@
 """Job REST adapter for the IBM Q Experience API."""
 
 import json
+import pprint
 
 from typing import Dict, List, Optional, Any
+from marshmallow.exceptions import ValidationError
 
 from .base import RestAdapterBase
+from .schemas.job import StatusResponseSchema
```

Comment:
Can we use a new file, instead of introducing the use `.schemas`? They are still not fully ready and probably will not make it into the release, and we'd rather avoid having to worry about them until they are in use (mostly as there might be quite a few tweaks due to the upgrade to marshmallow 3 and stabilization of the api). Perhaps `rest/validation.py`, along with some notes that the file is to be superseeded by `rest/schemas`?